compiler/src/dotty/tools/dotc/util/SimpleMap.scala

```diff
@@ -41,13 +41,13 @@ object SimpleMap {
   class Map1[K <: AnyRef, +V >: Null <: AnyRef] (k1: K, v1: V) extends SimpleMap[K, V] {
```

Comment:
Above, there is a definition `def contains(k: K): Boolean = apply(k) != null` which could use `ne` instead of `!=`.

Reply:
For null comparisons `!=` and `ne` produce exactly the same bytecode.